generators/boilerplate/templates/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function add(a, b) {
+  return a + b;
```

Comment:
yeah I would just opt to remove the `add` function and its tests, but keep the `module.exports` in.